connection.go

```diff
@@ -375,7 +375,6 @@ func (sc *snowflakeConn) queryContextInternal(ctx context.Context, query string,
 	rows := new(snowflakeRows)
 	rows.execResp = data
 	rows.sc = sc
-	rows.RowType = data.Data.RowType
```

Comment:
It's expected to have this line remove, as upstream did it in Feb 9th:
https://github.com/snowflakedb/gosnowflake/commit/9143c1f7011ec4acd4d1bfdac81b16cd1d18cc13#diff-25a7cf08cc8fc8eb57475835cc380524de2fd1215cda9c6915132ae86dbd80a7L340

Sees we previously accidentally picked it up in the "merged commit":
https://github.com/observeinc/gosnowflake/commit/c4d7bfe0bfcf0a9712e731a4ef24972a8d054c43#diff-25a7cf08cc8fc8eb57475835cc380524de2fd1215cda9c6915132ae86dbd80a7R350